docs/zero_to_hero_guide/quickstart.md

```diff
@@ -22,15 +22,23 @@ If you're looking for more specific topics like tool calling or agent setup, we
    - Download and unzip `Ollama-darwin.zip`.
    - Run the `Ollama` application.
 
-2. **Download the Ollama CLI**:
+1. **Download the Ollama CLI**:
    - Ensure you have the `ollama` command line tool by downloading and installing it from the same website.
 
-3. **Verify Installation**:
+1. **Start ollama server**:
+   - Open the terminal and run:
+      ```
+      ollama serve
+      ```
+
+1. **Verify Installation**:
    - Open the terminal and run:
      ```bash
-     ollama run llama3.2:1b
+     ollama run llama3.2:3b-instruct-fp16
      ```
-
+     Note:
+     1. The supported models for llama stack for now is listed in [here](https://github.com/meta-llama/llama-stack/blob/acbecbf8b3217d2594bad3eee8a322e16b8ee725/llama_stack/providers/remote/inference/ollama/ollama.py#L42-L69)
+     1. This command will notify ollama server that the model is available to use so the running order needs to be followed. Otherwise the server won't know the ollama model is available.
```

Comment:
Hm, this sentence is a bit confusing and I am not sure I quite understand it. Could you explain? Is it talking about the "ollama run" command?

Reply:
Here is my understanding on the start order:
1. start the ollama server with `ollama serve`
2. run the llama model via `ollama run [MODEL_VERSION]`
3. start the llama-stack via `llama stack run`

I am trying to tell we need to run the running order as above. I tried to run step 2 and then step 1 but llama stack will complain about model not found. 

Let me try to update that.

Reply:
Updated. Please review again

Reply:
@chuenlok So (1) is necessary. However, (2) and (3) in that order are not _as long as_ the user always called a `client.models.register()` before running inference.